README.md

```diff
@@ -1,30 +1,30 @@
 
-# o!StreamCompanion ![l](https://i.imgur.com/bXum3mS.png)
- 
-o!StreamCompanion is a program directed towards osu! streamers.
+### ![StreamCompanion](https://upload.hubza.co.uk/i/gate-b-logo-text.png)
```

Comment:
Please include this image in the repository itself. Create images folder at the root and place it there